src/store/bridge.ts

```diff
@@ -312,36 +309,32 @@ const actions = {
   async getHistory ({ commit }) {
     commit(types.GET_HISTORY_REQUEST)
     try {
-      commit(types.GET_HISTORY_SUCCESS, api.accountHistory)
+      commit(types.GET_HISTORY_SUCCESS, api.bridge.accountHistory)
     } catch (error) {
       commit(types.GET_HISTORY_FAILURE)
       throw error
     }
   },
-  saveHistory ({ commit }, history: BridgeHistory | null) {
-    if (!history || !history.id) {
-      return
-    }
-    api.saveHistory(history)
-  },
   setHistoryItem ({ commit }, historyItem: BridgeHistory | null) {
     commit(types.SET_HISTORY_ITEM, historyItem)
   },
+  async saveHistory ({ commit }, history: BridgeHistory) {
+    api.saveHistory(history)
+  },
   async clearHistory ({ commit }) {
-    const history = api.accountHistory
-    if (history && history.length) {
-      const filtered = history.filter(item => ![Operation.EthBridgeOutgoing, Operation.EthBridgeIncoming].includes(item.type))
-      storage.set('history', JSON.stringify(filtered))
-    }
+    api.bridge.clearHistory()
     commit(types.GET_HISTORY_SUCCESS, [])
   },
-  async getNetworkFee ({ commit, getters, rootGetters }) {
-    if (!getters.asset || !getters.asset.address) {
+  findRegisteredAsset ({ commit, getters, rootGetters }) {
+    return rootGetters['assets/registeredAssets'].find(item => item.symbol === getters.asset.symbol)
```

Comment:
please find by token address. It's very important! I created 3 VAL tokens

Reply:
I'll add TODO there. I'm working with history and don't have addresses there (only symbols), so we should fix it later in substrate lib probably.